doc/user/content/installation/install-on-gcp/appendix-deployment-guidelines.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Appendix: GCP deployment guidelines"
+description: "GCP environment setup/deployment guidelines"
+menu:
+  main:
+    parent: "install-on-gcp"
+    identifier: "gcp-deployment-guidelines"
+    weight: 40
+---
+
+## Recommended instance types
+
+As a general guideline, we recommend:
```

Comment:
Added details for the disk enablement.

https://preview.materialize.com/materialize/32299/self-managed/v25.1/installation/install-on-gcp/appendix-deployment-guidelines/